base/sparse/sparsematrix.jl

```diff
@@ -1405,7 +1405,6 @@ conj!(A::SparseMatrixCSC) = (broadcast!(conj, A.nzval, A.nzval); A)
 
 # TODO: The following definitions should be deprecated.
 ceil{To}(::Type{To}, A::SparseMatrixCSC) = ceil.(To, A)
-floor{To}(::Type{To}, A::SparseMatrixCSC) = floor.(To, A)
```

Comment:
Depending on how you interpret the docstring? `floor(T, x) converts the result to type T, throwing an InexactError if the value is not representable.`